### PR TITLE
Adding dependencies in kytos.json

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

File: napps/kytos/of_topology/kytos.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "name": "of_topology",
   "description": "Keeps track of links between hosts and switches. Returns a JSON with network topology using the endpoint /kytos/topology.",
   "version": "0.1.1",
-  "napp_dependencies": [],
+  "napp_dependencies": ["kytos/of_core", "kytos/of_l2ls", "kytos/of_lldp"],
```
Comment (beraldoleal) [inline napps/kytos/of_topology/kytos.json]: of_topology should not depend on of_l2ls.

File: napps/kytos/web_topology_layout/kytos.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "name": "web_topology_layout",
   "description": "Manage endpoints related to the web interface settings and layout.",
   "version": "0.1.0",
-  "napp_dependencies": [],
+  "napp_dependencies": ["kytos/of_core"],
```
Comment (diraol) [inline napps/kytos/web_topology_layout/kytos.json]: This NApp depends on almost all other NApps as far as I remember, doesn't it?

Comment (macartur) [inline napps/kytos/web_topology_layout/kytos.json]: This NApp only expose 2 routes to save and load topologies from file.

**Comment by diraol** [comment]
This PR depends on kytos/kytos-napps#275 in order to pass on the CI tests.

**Pull request merged**